src/containerapp/azext_containerapp/_params.py

```diff
@@ -485,10 +485,10 @@ def load_arguments(self, _):
         c.argument('name', name_type, metavar='NAME', id_part='name', help=f"The name of the Container Apps Job. A name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, cannot have '--', and must be less than {MAXIMUM_CONTAINER_APP_NAME_LENGTH} characters.")
         c.argument('cron_expression', help='Cron expression. Only supported for trigger type "Schedule"')
         c.argument('image', help="Container image, e.g. publisher/image-name:tag.")
-        c.argument('replica_completion_count', type=int, options_list=['--replica-completion-count', '--rcc'], help='Number of replicas that need to complete successfully for execution to succeed.')
-        c.argument('replica_retry_limit', type=int, help='Maximum number of retries before the replica fails.')
-        c.argument('replica_timeout', type=int, help='Maximum number of seconds a replica can execute.')
-        c.argument('parallelism', type=int, help='Maximum number of replicas to run per execution.')
+        c.argument('replica_completion_count', type=int, options_list=['--replica-completion-count', '--rcc'], help='Number of replicas that need to complete successfully for execution to succeed, by default 1')
+        c.argument('replica_retry_limit', type=int, help='Maximum number of retries before the replica fails, by default 0')
```

Comment:
Are these `by default $value` redundant with the `default=`? 
Sample setting:
`c.argument('tail', help="The number of past logs to print (0-300)", type=int, default=20)`
test with `az containerapp logs show -h` :
`--tail                         : The number of past logs to print (0-300).  Default: 20.`

Reply:
added the default=$value and removed redundant text except for when default=0 the default text would not properly display so left that text in